.github/workflows/build-test.yaml

```diff
@@ -1153,98 +1189,14 @@ jobs:
           cluster-id: ${{ steps.create-cluster.outputs.cluster-id }}
 
 
-  validate-gke-install:
```

Comment:
removed this as we're testing on CMX GKE - any concerns?

Reply:
nope